### DISCUSSION TITLE
`dnt=1` and cross-site tracking cookies - How to avoid the need for an opt-in cookie request?

### DISCUSSION THREAD

**Discussion by tvercaut** [vimeo/player.js#889] [Q&A]
Hi,

I am trying to embed a vimeo histed video in a webpage without needing an opt-in cookie policy. I understood that the "do not track", `dnt=1` parameter was designed for that purpose:
https://complianz.io/embedding-vimeo-videos-privacy-friendly/

However, firefox reports that the vimeo player makes use of cross-site tracking cookies despite the DoNotTrack request. As one can see on https://complianz.io/embedding-vimeo-videos-privacy-friendly/ when the browser is configured to block cross-site tracking, the console reports the following:
```
Request to access cookie or storage on “https://player.vimeo.com/video/643915247?title=0&byline=0&portrait=0&badge=0&quality=1080p&dnt=1” was blocked because it came from a tracker and content blocking is enabled.
```

Am I misunderstanding the dnt=1` parameter or is there something else that needs to be done to prevent vimeo from requesting such cross-site tracking cookies.

Many thanks,
Tom

**Comment by DATEx2** [comment]
17 months later and this issue is still very annoying and no body cares :(

**Reply by robinp** [reply]
Without more details, hard to tell, but at least now FF reports partitioned storage (as it should) and Vimeo indeed sets only the reduced set of cookies.
